src/main/java/gumbo/parser/Parser.java

```diff
@@ -0,0 +1,124 @@
+package gumbo.parser;
+
+import gumbo.commands.*;
+import gumbo.exceptions.IllegalValueException;
+import gumbo.tasks.Deadline;
+import gumbo.tasks.Event;
+import gumbo.tasks.Todo;
+
+/**
+ * Interprets user input and creates the corresponding {@code Command} objects to execute user instructions.
+ */
+public class Parser {
+
+    /**
+     * Parses the user input and returns the corresponding {@code Command} object.
+     *
+     * @param fullCommand The full string of user input.
+     * @return A {@code Command} object representing the user's request.
+     */
+    public static Command parse(String fullCommand) {
+        if (fullCommand.equals("bye")) {
```

Comment:
The current implementation uses a series of if-else statements. Would it be cleaner to switch to a switch-case statement improve readability and organization?